SwissTransferFeatures/SettingsView/SettingsView.swift

```diff
@@ -138,8 +139,8 @@ public struct SettingsView: View {
                 .settingsCell()
 
                 Button {
-                    @InjectService var reviewManager: ReviewManageable
-                    reviewManager.requestReview()
+                    guard let appStoreReviewURL = Constants.appStoreReviewURL else { return }
```

Comment:
If the user has downloaded the app through TestFlight, they won't able to give a feedback on the app store.
Should we add a condition and use the ReviewManager for the people coming from TestFlight?

Reply:
Hiding the cell if isRunningInTestflight (same on Infomaniak Mail)